charts/postgres-operator/templates/customrresourcedefinition.yaml

```diff
@@ -54,6 +54,303 @@ spec:
   scope: Namespaced
   subresources:
     status: {}
+  validation:
+    openAPIV3Schema:
+      required:
+        - kind
+        - apiVersion
+        - metadata
+        - spec
+      properties:
+        kind:
+          enum:
```

Comment:
Looks like there should be also `type`

```yaml
apiVersion:
  type: string
  enum:
    - acid.zalan.do/v1
```
otherwise K8S is complaining:

```
spec.validation.openAPIV3Schema.properties[apiVersion].type: Invalid
value: "": must be string, spec.validation.openAPIV3Schema.properties[apiVersion].type:
```